README.md

```diff
@@ -1564,6 +1567,110 @@ for domain in domains:
     print(domain)
 ```
 
+### Create Static brcode
+
+Dynamic Brcodes store account information in the form of a Pix Key and can easily be used to create Pix transactions.
+
+```python
+import starkinfra
+
+brcodes = starkinfra.staticbrcode.create(
+    brcodes=[
+        starkinfra.StaticBrcode(
+            name="Jamie Lannister",
+            key_id="+5511988887777",
+            amount=100,
+            reconciliation_id="123",
+        )
+    ]
+)
+
+for brcode in brcodes:
+    print(brcode)
+```
+
+### Query Static brcode
+
+You can query multiple Static brcodes according to filters.
+
+```python
+import starkinfra
+
+brcodes = starkinfra.staticbrcode.query(
+    limit=1,
+    after="2022-06-01",
+    before="2022-06-30",
+    uuids=["5ddde28043a245c2848b08cf315effa2"]
+)
+
+for brcode in brcodes:
+    print(brcode)
+```
+
+### Get a Static brcode
+
+After its creation, information on a Static brcode may be retrieved by its.
+
+```python
+import starkinfra
+
+brcode = starkinfra.staticbrcode.get("5ddde28043a245c2848b08cf315effa2")
+
+print(brcode)
+```
+
+### Create Dynamic brcode
+
+Dynamic Brcodes store account information in the form of a Pix Key and can easily be used to create Pix transactions.
```

Comment:
Dynamic BR Codes store account information via a an URL that can be dynamically changed by you so as to update payment information. They can easily be used to receive strongly reconciled Pix transactions.